app/views/devise/registrations/new.html.erb

```diff
@@ -29,6 +29,11 @@
             <%= f.password_field :password_confirmation, autocomplete: "new-password", class: "form-control" %>
           </div>
 
+          <div class="form-group">
+            <%= f.label  :consent_news_email, "Would you like to opt-in to email updates from Forward Democracy ?", class: "mb-0" %>
+            <%= f.check_box :consent_news_email, class: "form-control" %>
```

Comment:
Can we make this default to checked?  Otherwise most people won't check it, and then we miss the majority of the benefit of this PR.

Reply:
My understanding is that having it opt-in, ie unchecked by default, is part of the same legal principle that says it's needed. It makes communications marketing harder, but makes sense from the user perspective.

Reply:
(Ie a large part of the benefit of the Issue is making the product better compliant)

Reply:
my understanding too, i.e. it can't be pre-ticked.

specifically because the form itself is not a newsletter signup. if it was a newsletter signup you wouldn't even need the opt-in, because that's explicit.

as an aside, that's one of the issues with the action network form, it forces an opt-in on what is ONLY a newsletter signup. very frustrating.

Reply:
also as an aside, #244 was partly to help maximise 'conversions', iirc

Reply:
Okay then, sounds like we shouldn't do this change.